utils/DSM_FOR_SPECFEM3D/routines_for_conversion_geogr_cart/geogr2cart.f90

```diff
@@ -102,7 +102,7 @@ subroutine  compute_rotation_matrix(rotation_matrix,lon_center_chunk,lat_center_
   end subroutine compute_rotation_matrix
 
 !--------------------------------------------------------------------------------------------------------------------------------------------------------
-! 3D rotation matrix with axis  "axe" and angle theta (d°)
+! 3D rotation matrix with axis  "axe" and angle theta (d?)
```

Comment:
Thanks! I fixed all of them.

On 01/22/2015 04:56 AM, Elliott Sales de Andrade wrote:
> In
> utils/DSM_FOR_SPECFEM3D/routines_for_conversion_geogr_cart/geogr2cart.f90 <https://github.com/geodynamics/specfem3d/pull/366#discussion_r23354308>:
>
>> @@ -102,7 +102,7 @@ subroutine  compute_rotation_matrix(rotation_matrix,lon_center_chunk,lat_center_
>>    end subroutine compute_rotation_matrix
>>
>>  !--------------------------------------------------------------------------------------------------------------------------------------------------------
>> -! 3D rotation matrix with axis  "axe" and angle theta (d°)
>> +! 3D rotation matrix with axis  "axe" and angle theta (d?)
>
> And here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d/pull/366/files#r23354308>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr